VocaDbWeb/New/components/Folder/Folder.tsx

```diff
@@ -0,0 +1,49 @@
+import { DiscussionFolderContract } from '@/types/DataContracts/Discussion/DiscussionFolderContract';
+import { Grid, Title } from '@mantine/core';
+import useStyles from './Folder.styles';
+import { ThemeIcon } from '@mantine/core';
+import { IconBook2 } from '@tabler/icons-react';
+
+interface FolderProps {
+	folder: DiscussionFolderContract;
+}
+
+export function Folder({ folder }: FolderProps) {
+	const styles = useStyles();
+	return (
+        <>
+            <Title mt="md" mb="xs" order={5} className={styles.classes.folderName}>
+                <ThemeIcon variant="filled" style={{ display: "inline-flex", marginRight: "10px" }}><IconBook2 size="1rem" style={{ display: "inline-flex" }} /></ThemeIcon>{folder.name}
```

Comment:
I'm not sure if display: "inline-flex" is necessary here? You can also set marginRight using the mr property (https://mantine.dev/styles/style-props/#supported-props)